app/controllers/people_controller.rb

```diff
@@ -0,0 +1,27 @@
+class PeopleController < ApplicationController
+
+  def index
+    @all_people = Person.all
+
+    render :index
+  end
+
+  def show
+    @person = Person.find(params[:id])
+
+    @person_id =        @person[:id]
+    @person_name =      @person[:name]
+    @person_fav_genre = @person[:fav_genre]
+
+    render :person
+  end
+
+  def all_tasks
+    @person =      Person.find(params[:id])
+    @person_name = @person.name
+    @all_tasks =   @person.tasks
+
+    render :all
```

Comment:
This could be a misconceiving name for your view since its coming from the People controller, but it is not all people it is all associated tasks. Recommend renaming for clarity.